mainPage/alpha/normativeTypes/normativeTypes.html

```diff
@@ -138,8 +140,9 @@ <h2 class="nocount">Abstract</h2>
       <h2 class="nocount">Status of this Document</h2>
       <!-- Statement about why this version exists -->
       
-      <p>This is the 16 Mar 2015 version of the Normative Types document.
-       This version updates the definitions of time_t, control_t, display_t,
+      <p>This is the 16 Mar 2015 version of the Normative Types document, with the addition
+       of minor clarifications to the dscription of NTTable. The 16 Mar 2015 version
```

Comment:
1. Typo.
2. I would prefer the 12-Aug version to start with the sentence "This is the 12 Aug 2015 version..." else it looks like a cut-n-paste mistake.